### DISCUSSION TITLE
malfunzionamento dei moduli

### DISCUSSION THREAD

**Discussion by fxlduto** [iisgiua/giuaschool#617] [Richieste di aiuto]
salve, quando vado a compilare il modulo di evacuazione ma comunque in generale i moduli mi dà errore 500

**Comment by trinko** [comment]
Vedi [Come fare una segnalazione di errore](https://github.com/iisgiua/giuaschool/discussions/576)

**Comment by ghost** [comment]
@spinalucx Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
